### ISSUE TITLE
Prefill the demo website

### ISSUE THREAD

**Issue by Jeroenvb3** [LibrePhotos/librephotos#741]
**Describe the enhancement you'd like**
Prefill the demo website with photos.

**Describe why this will benefit the LibrePhotos**
Currently you can view the UI in the demo, but it lacks context if there are no photos to play around with.

**Comment by loulou91** [comment]
You can upload photos and play with...

**Comment by DigitalWarrior** [comment]
@Jeroenvb3 What demo where you using?

**Comment by Nerodacles** [comment]
> @Jeroenvb3 What demo where you using?

It *has* to be `demo2` because that is the only instance that gets "updated" to the latest `dev` version.

**Comment by Jeroenvb3** [comment]
@DigitalWarrior I was using demo2, indeed.

**Comment by RobP-git** [comment]
I agree with OP. As a first time viewer of the project, I went to demo2 to look at it and there is nothing to look at. I'm not about to upload my own personal files to a public demo!

**Comment by DigitalWarrior** [comment]
demo1.librephotos.com has preloaded images. Every time the demo1 pulls the latest images the db is reloaded with prior scan data. 
demo2.librephotos.com does not have preloaded images. So it is like a fresh install, just the user is preset. 

Demo 1 runs the stable branch
Demo 2 runs the dev branch

Are you suggesting that there be a demo 3 &  4. 
Two dev with one having preloaded scan?
Two stable with one having preloaded scan?

**Comment by RobP-git** [comment]
I wasn't aware there was a demo1.  Please update README.md to reflect this, that's probably all thats needed.

edit: I just had a look at demo1. It could use some improvement as a demo to show off your best features. As a first time viewer who isn't familiar with the project, the demo1 appears half broken and the images are all faces not actual photos. First impressions count.
